### DISCUSSION TITLE
How ElementNode implements functions similar to the decorate function of DecoratorNode. I want to customize code-block but I don’t have any ideas

### DISCUSSION THREAD

**Discussion by frankdiw** [facebook/lexical#5843] [Q&A]
<img width="1095" alt="image" src="https://github.com/facebook/lexical/assets/17613480/f32c1d2f-c65a-4c2e-8f28-58bc7007bc5b">
I want to implement the style as shown above, but ElementNode seems to be unable to implement it.
@ivailop7 If possible, please provide me with a suggestion, thank you！

**Comment by ivailop7** [comment]
A good example to add interactivity to an elementNode is FloatingLinkPlugin or the CodeActionsMenu. Otherwise the decorate equivalent is the createDOM.

**Comment by ivailop7** [comment]
If it's just the styling, you can apply a custom CSS class in the createDOM and define it as a normal CSS class.

**Reply by frankdiw** [reply]
Thank you for your reply. Not only the style is modified, but also the html tag needs to be added. CodeActionsMenu is targeted at the current code-block. I want to add to all of them.
